lib/jmespath/nodes.rb

```diff
@@ -5,10 +5,6 @@ class Node
       def visit(value)
       end
 
-      def hash_like?(value)
-        Hash === value || Struct === value
```

Comment:
I believe this may be a breaking change as `Struct` does not implement `to_hash`.

Reply:
yes, since Struct has no #to_hash, I changed the places which used this method to either call #to_hash or handle the Struct, handling both cases. by refactoring those there were no longer any calls to this method, so I removed it.

Reply:
Gotcha - agree it looks like Struct is being handled correctly where this was used.